packages/react-native-bridge/package.json

```diff
@@ -1,31 +1,31 @@
 {
-  "name": "@wordpress/react-native-bridge",
```

Comment:
Based on https://github.com/WordPress/gutenberg/pull/28634#discussion_r568479085, it looks like it may be a good idea for us to manually change the `package.json` files back to tabs in our editor.

Reply:
I think I am mistaken on this. This appears to be the inverse of what I said. I think these changes should be left as is.